### ISSUE TITLE
Hotkey not being saved (back to default after instance restart)

### ISSUE THREAD

**Issue by Chr1s70ph** [leokraft/QuickKey#36]
This has something to do with the default directory the program is installed to.

By default, you need admin rights to change files in the directory. So changing the Hotkey in the settings only works for that instance of the program. The changes wont save. 

saving the file in notepad without elevated permissions even leads to this error:
![image](https://user-images.githubusercontent.com/29735603/127678611-b0f2ceba-1aa8-46a3-8ba9-16eb595fa03b.png)

A workaround for now is to open the `config.ini` in an editor with elevated permissions and safe the file that way.

Moving the config to `%appdata%` would be the better place tho in the long term, because the program can edit the files there, without needing elevated permissions.

So in layman's terms: the application cant save the changes to the config file.

**Comment by leokraft** [comment]
Thank you for your issue!
I definitely agree with you on the causes and workarounds/fixes.

### TODO:

1. create default config
2. move user config to AppData
3. implement check for user config and use default config as fallback
